glue/egmde.launcher

```diff
@@ -18,7 +18,12 @@ then
     let x11_display+=1
   done
 
-  export MIR_SERVER_X11_DISPLAY_EXPERIMENTAL=${x11_display}
+  if $SNAP/bin/egmde --help 2>&1 | grep '\--x11-display-experimental' > /dev/null
+  then
+    export MIR_SERVER_X11_DISPLAY_EXPERIMENTAL=${x11_display}
+  else
+    export MIR_SERVER_X11_DISPLAY=${x11_display}
```

Comment:
Hmm, might be simpler and less error prone to set both with a comment explaining that experimental can eventually be removed

Reply:
That doesn't work. The boost options code throws if it encounters a MIR_SERVER_* env variable it doesn't recognise